src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -135,6 +154,12 @@ private void SetStorageServiceAndTaskHubClient(out AzureStorageOrchestrationServ
         private void Initialize(out AzureStorageOrchestrationService orchestrationService, out TaskHubClient taskHubClient, string connectionStringKey = null, string taskHubName = null)
         {
             CheckAssemblies();
+
+            if (this.BackendType != BackendType.AzureStorage)
+            {
+                throw new CliException($"The {this.BackendType} storage provider for Durable Functions is not yet supported by this command.");
```

Comment:
TBD on whether we want to try supporting these other backends as part of this PR.